Develop

SofiaMalts opened the pull request:
Sprint_4_project_pullRequest_1

SofiaMalts committed:
Sprint_4_version_1

SofiaMalts committed:
readme_update

pomegranate282922 commented on src/main/java/org/example/Main.java:
```diff
@@ -0,0 +1,7 @@
+package org.example;
+
+public class Main {
+    public static void main(String[] args) {
```
Это ненужный класс. Нужно убрать.

pomegranate282922 commented on src/test/java/PageObjectsPackage/CreateOrderPageObjects.java:
```diff
@@ -0,0 +1,201 @@
+package PageObjectsPackage;
```
Пакеты именуем строчными буквами без camelCase

pomegranate282922 commented on src/test/java/Tests/CreateOrder.java:
```diff
@@ -0,0 +1,79 @@
+package Tests;
+import PageObjectsPackage.HomePageObjects;
```
Тоже нужно поправить нейминг

pomegranate282922 commented on src/test/java/Tests/CreateOrder.java:
```diff
@@ -0,0 +1,79 @@
+    public void createOrderFromUpperButton(){
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.get("https://qa-scooter.praktikum-services.ru/");
```
Урлы лучше выносить как константы. Так их проще переиспользовать и изменять

pomegranate282922 commented on src/test/java/Tests/ExpandImportantQuestionTest.java:
```diff
@@ -0,0 +1,51 @@
+    @Parameterized.Parameters
+    public static Object[][] getData() {
+        return new Object[][] {
+                {"Сколько это стоит? И как оплатить?", "Сутки — 400 рублей. Оплата курьеру — наличными или картой."},
```
Нужно проверить все вопросы и ответы.

pomegranate282922 reviewed (commented):
Все очень хорошо реализовано. Очень понравилась page object model.
Есть небольшие замечания к неймингу и тестам. Прошу их поправить.

SofiaMalts committed:
Sprint_4_version_2
- Удален класс Main
- Отредактированы названия пакетов с POM и тестами. Немного изменена структура пакетов
- URL вынесен как константа
- В параметры добавлены недостающие вопросы и ответы.

pomegranate282922 reviewed (approved):
Зачёт!
